### DISCUSSION TITLE
请问一下挂载和彩云是应该填写哪个COOKIE呢？

### DISCUSSION THREAD

**Discussion by D3-3109** [libsgh/PanIndex#319] [Q&A] [answered]
刚把面板部署到railway，然后尝试挂载和彩云。
填写了手机号之后卡在了COOKIE这一步。
尝试填写了ORCHES-C-TOKEN中的内容，保存后显示登录失败。
翻看LOG发现应该是验证失败
ERRO[2022-05-19 21:17:02] <nil>                                         function=SyncAccountStatus line=315
ERRO[2022-05-19 21:17:02] [yun139] 移动CDN auth login fail, api return :   function=SyncAccountStatus line=316

**Comment by libsgh** [comment, marked as answer]
完整的cookie，访问首页登陆后，用开发工具查看Request Header中的Cookie字段，复制

**Reply by icechang87** [reply]
Cookie字段全部复制，还是显示失败。具体复制哪一段，有大佬告诉一下嘛？
